README.md

```diff
@@ -0,0 +1,56 @@
+# ApidAnalytics
```

Comment:
I ran go fmt, go vet and the race checker tool.
1. go fmt - fixed the formatting
2. The race checker tool found a data race related to the way I was creating cache. I fixed that already.
3. There is another one it finds in buffer manager while trying to write and close a file. It thinks the file handler will cause a data race. I ran through the logic again and I am sure that will not be an issue since I am managing access to that variable by waiting on a channel.